tasks/qiime-alpha-beta-diversity/main.py

```diff
@@ -23,7 +23,8 @@ def diversityCoreMetricsPhylogeneticSample(
     tablePath: Path,
     samplingDepth: int,
     metadataPath: Path,
-    outputDir: Path
+    outputDir: Path,
+    threads
```

Comment:
is there any special reason why are we missing type for threads here?

Reply:
Types will need to be added here, but lets do that in a different task for integrating mypy github action on this repo to check for typing errors.